core/src/main/java/com/mapzen/android/core/MapzenManager.java

```diff
@@ -48,16 +49,24 @@ public static MapzenManager instance(Context context) {
    */
   private MapzenManager(Context context) {
     final Resources resources = context.getResources();
-    int id = resources.getIdentifier(API_KEY_RES_NAME, API_KEY_RES_TYPE, context.getPackageName());
-    if (id > 0) {
-      apiKey = resources.getString(id);
+    if (resources != null) {
+      int id = resources.getIdentifier(API_KEY_RES_NAME, API_KEY_RES_TYPE,
+          context.getPackageName());
+      if (id > 0) {
+        apiKey = resources.getString(id);
+      }
     }
   }
 
   /**
    * Returns the currently active API key stored by this class.
```

Comment:
Good catch! Added in https://github.com/mapzen/android/pull/299/commits/2675af43ae1a6f99cdd20f1dc1dbe9185ae20f76.